src/openfl/_internal/renderer/dom/DOMMaskManager.hx

```diff
@@ -31,7 +31,7 @@ class DOMMaskManager extends AbstractMaskManager {
 	}
 	
 	
-	public override function pushMask (mask:DisplayObject):Void {
+	public function pushMask (mask:DisplayObject):Void {
 		
 		// TODO: Handle true mask shape, as well as alpha test
```

Comment:
Is this comment still valid or isn't this exactly describing what you fixed now?

Reply:
Well, it's still valid for the DOM things